### PR TITLE
Portfolio- Project: Mobile menu

### PR DESCRIPTION
**Pull request by theodoku**
**1. Created a new branch
2. Add a close-btn image
3. Style nav-items and nav-links
4. Add a script.js file and created eventListeners for openMobileMenu and closeMobileMenu**

### CODE REVIEW

**Review by nahnahdev — COMMENTED** [review]
Hi @theodoku and @ritikarawat220 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is:
- [ ] Missing the javascript linters. From the screenshot below, you can see that the only linters that run were the HTML and CSS linters. As stated in your project requirements, you are to add the [JavaScript + HTML + CSS linter](https://github.com/microverseinc/linters-config/tree/master/html-css-js) so kindly do this :wink:.

image

- [ ]  You are also missing some headers in your README file, in the screenshot below, you can see that after the `Tech Stack`, you're supposed to have the `Key Features `section. But you don't have that section in your README file. So kindly add this as well so your project can be reviewed :wink:. This is necessary because every section of the `Microverse README template `is compulsory with the exception of the `FAQ's section `:100: 
image



- Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -546,8 +545,66 @@ button {
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 80%;
+  z-index: 100;
+}
+
+.nav_item {
+  padding: 50px 30px;
+}
```
Comment (nahnahdev) [inline style.css]: - [x] Great job so far :muscle: However I noticed here that the padding you applied to the list items was too big making the design look slightly different from the Figma Design. So kindly try to fix this and try to stick to the Figma design as much as possible :wink:
- Figma Design
image
- Your Design
image

File: style.css [hunk 2]
```diff
@@ -546,8 +545,66 @@ button {
...
+.header_wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 3rem;
+  align-content: center;
+
+  /* margin-left: 15px;
+  margin-right: 15px; */
+}
```
Comment (nahnahdev) [inline style.css]: - [ ] Kindly avoid leaving commented pieces of code in your repo as it makes your  codebase unclean :+1:

**Review by nahnahdev — CHANGES_REQUESTED** [review]
Hello @theodoku @ritikarawat220, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [ ] Good use of GitHub flow :+1: 
- [ ] Menu works perfectly :100: 
- [ ] Linters properly setup and passing :100: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (theodoku) [reply style.css]: worked on

File: style.css (hunk 2 above)
Comment (theodoku) [reply style.css]: removed uncommented codes

**Review by topeogunleye — APPROVED** [review]
### Hi @theodoku, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**